src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.PlatformNotSupported.cs

```diff
@@ -269,7 +269,17 @@ public static class Sse
         /// <summary>
         /// __m128 _mm_load_ps (float const* mem_address)
         /// </summary>
-        public static unsafe Vector128<float> LoadAligned(float* address) { throw new PlatformNotSupportedException(); }
+        public static unsafe Vector128<float> LoadAlignedVector128(float* address) { throw new PlatformNotSupportedException(); }
+
+                /// <summary>
+        /// __m128 _mm_loadh_pi (__m128 a, __m64 const* mem_addr)
+        /// </summary>
+        public static unsafe Vector128<float> LoadHigh(Vector128<float> value, float* address) { throw new PlatformNotSupportedException(); }
```

Comment:
I thought we weren't exposing any of the __m64 intrinsics?

Reply:
It is not MMX intrinsic:
```
__m128 _mm_loadh_pi (__m128 a, __m64 const* mem_addr)
```
It loads to `xmm` register

Reply:
Ah, right.

Reply:
Looks like its just a confusing signature on the C/C++ side.

The Instruction manual indeed says it only operates on m64 and doesn't deal with _m64 registers at all:
![image](https://user-images.githubusercontent.com/10487869/34645535-0997fc8a-f305-11e7-8bc1-6b051d2b22fb.png)

Reply:
@4creators Loading data 64-bit by 64-bit into XMM makes no sense on modern CPUs.

Reply:
(it also doesn't have the "causes a transition to MMX technology" statement)

Reply:
> Loading data 64-bit by 64-bit into XMM is useless on modern CPUs.

In contrary actually. You may want lower 64 bits for one memory location and upper 64 bits from another memory location placed far apart from each other. In such situations this function is a perfect tool and there are many cases where such loads are necessary.

Reply:
@fiigii 
As you can see above  it is available even for AVX512 ISA

Reply:
@4creators, I think the point was that, on modern CPUs there are better instructions for doing this, and even on older CPUs, two loads and a shuffle or Unpack are likely better

Reply:
Okay, let's keep these APIs.

Reply:
We can remove useless APIs later if there is.

Reply:
> We can remove useless APIs later if there is.

I think being conservative in deleting APIs which are available in given ISA is the best approach.

Reply:
@fiigii, if we are keeping it (and we probably should, since it is a regular SSE instruction, and not an MMX Interop instruction), we should also be exposing the `StoreHigh` and `StoreLow` counterparts (for `movhps m64, xmm` and `movlps m64, xmm`)

Reply:
(and the same for Sse2)

Reply:
> we should also be exposing the StoreHigh and StoreLow counterparts

Thanks, will do.